### ISSUE TITLE
Hearts not refilling upon death

### ISSUE THREAD

**Issue by Georg255** [Cazsius/Spice-of-Life-Carrot-Edition#58]
Not sure whether this is 1,12.2 SOLC or another mod but whenever I die (which happens a lot in my pack), the hearts refuse to regen anymore.  For instance, in my current world, I have a max of 5 hearts (started with 3 max).  Hearts won't regen (have max food/a lot of saturation) past three.  Don't know what to do.

EDIT: If it helps, the following mods are affecting the health cap: SOLC, Father Toast's Natural Absorption, Tinker's (the display).  Possibly AttributeFix, too.

**Comment by juliand665** [comment]
If you'd like to accelerate progress on this, it would help if you could reproduce this in as minimal a mod set as possible (perhaps even just SOLC is enough?).

**Comment by Georg255** [comment]
I'll give it a go with just SOLC and report back.

EDIT: No dice.  It didn't happen again with just SOLC.  I got killed by something that gave me potion effects in the main pack by a Grimoire of Gaia monster.  Does that help?

DOUBLE EDIT: Confirmed that it's not your mod.  Something else in the pack is screwing it up.  Sorry to bother you.

TRIPLE EDIT:  If you're still watching this, I'm 80% certain the culprit is Father Toast's Natural Absorption.  The bug has yet to show after I removed it.
